buildtest/utils/command.py

```diff
@@ -54,27 +116,26 @@ def execute(self):
         # Use updated command with executable and remainder (list)
         cmd = [executable] + args
 
-        # open the process for writing
-        process = subprocess.Popen(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        returncode = process.poll()
-
-        # Iterate through the output
-        while returncode is None:
-
-            out = self.decode(process.stdout.readline())
-            err = self.decode(process.stderr.readline())
-
-            # Append output and error
-            if out:
-                self.out.append(out)
-            if err:
-                self.err.append(out)
+        # Capturing provides temporary output and error files
```

Comment:
Why are these changes showing up here when I already made them in another PR? I wrote these lines...

Reply:
I was testing something, this was an accident my bad ill undo this, ill copy the file from devel since i didnt make any change to the file.